tests/e2e/kserve_test.go

```diff
@@ -206,3 +210,63 @@ func (c *KserveTestCtx) validateDefaultCertsAvailable(t *testing.T) {
 	g.Expect(ctrlPlaneSecret.Type).Should(Equal(defaultIngressSecret.Type))
 	g.Expect(defaultIngressSecret.Data).Should(Equal(ctrlPlaneSecret.Data))
 }
+
+func (c *KserveTestCtx) ValidateKserveOwnsFeatureTrackers(t *testing.T) {
+	g := c.NewWithT(t)
+
+	fts := []string{
+		// c.ApplicationNamespace + "-kserve-external-authz",
```

Comment:
This commented out reference, and the other ones that are commented out in the other test below, are commented because they don't get deployed on OpenShift CI, because we don't install authorino there.